server/Server.ts

```diff
@@ -169,6 +169,7 @@ class Server {
     this.app.use('/proto', express.static(path.join(__dirname, 'static/proto')));
     this.app.use('/style', express.static(path.join(__dirname, 'static/style')));
     this.app.use('/worker', express.static(path.join(__dirname, 'static/worker')));
+    this.app.use('/assets', express.static(path.join(__dirname, 'static/assets')));
```

Comment:
This fixes accessing the model file that would not be available to the web server when bundled for production